mod-credits/src/main/java/gg/solarmc/loader/credits/CreditsLoader.java

```diff
@@ -22,27 +22,31 @@
 import gg.solarmc.loader.data.DataLoader;
 import gg.solarmc.loader.Transaction;
 import gg.solarmc.loader.impl.SQLTransaction;
+import org.jooq.DSLContext;
 
 import java.math.BigDecimal;
 
 import static gg.solarmc.loader.schema.tables.Credits.CREDITS;
 
 class CreditsLoader implements DataLoader<Credits> {
 
+	private final BigDecimal defaultBigDecimalBalance;
+
+	public CreditsLoader(CreditsConfig configuration) {
+		defaultBigDecimalBalance = BigDecimal.valueOf(configuration.defaultBalance());
```

Comment:
you can move BigDecimal.valueOf to CreditsKey#createLoader, and thereby avoid code in the constructor

Reply:
Fixed this